deploy/manifests/deployment.yaml

```diff
@@ -34,8 +34,8 @@ spec:
         - name: nginx
           mountPath: /etc/nginx
       containers:
-      - image: ghcr.io/nginxinc/nginx-kubernetes-gateway:edge
-        imagePullPolicy: Always
+      - image: ghcr.io/nginxinc/nginx-kubernetes-gateway:0.4.0
```

Comment:
Do we also need to update the provisioner image tag and pull policy? @pleshakov

Reply:
> Do we also need to update the provisioner image tag and pull policy? @pleshakov

I think it makes sense. As well as updating conformance-related configuration, so that the images are taken not edge, but tag. Like this one https://github.com/nginxinc/nginx-kubernetes-gateway/blob/68cc2d243e9a4b6c5fb7696c90661fc17d7b0773/conformance/Makefile#L1

Reply:
We should add these steps to the release process doc